docker-compose.yml

```diff
@@ -0,0 +1,55 @@
+version: '2.2'
+services:
+  es01:
+    #uncomment after elasticsearch upgrade
+    #image: docker.elastic.co/elasticsearch/elasticsearch:7.10.0
+    image: docker.elastic.co/elasticsearch/elasticsearch:6.8.3
+    container_name: es01
+    environment:
+      - node.name=es01
+      - cluster.name=es-docker-cluster
+      - discovery.type=single-node
+      - bootstrap.memory_lock=true
+      - "ES_JAVA_OPTS=-Xms512m -Xmx512m"
+      - xpack.security.enabled=false
+      - xpack.monitoring.enabled=false
+      - xpack.graph.enabled=false
+      - xpack.watcher.enabled=false
+      - xpack.ml.enabled=false
+    ulimits:
+      memlock:
+        soft: -1
+        hard: -1
+    volumes:
+      - data01:/usr/share/elasticsearch/data
+    ports:
+      - 9200:9200
+    networks:
+      - elastic 
+  # uncomment to run a kibana instance 
+  # kib01:
+    # image: docker.elastic.co/kibana/kibana:6.8.3
+    # container_name: kib01
+    # ports:
+      # - 5601:5601
+    # environment:
+      # ELASTICSEARCH_URL: http://es01:9200
+      # ELASTICSEARCH_HOSTS: http://es01:9200
+    # networks:
+      # - elastic
+  postgres_db:
+    image: postgres:11.6
```

Comment:
@coverbeck 

11.8 or something?

Reply:
Yeah, we're apparently on 11.8 in prod -- PG auto updates.